### PR TITLE
PR triggering on code review

### PR DESCRIPTION
**Pull request by viovanov**
This should work for forks as well.

### CODE REVIEW

File: .github/workflows/pull-request-queue.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  enqueue:
+    if: github.event.pull_request.base.ref == 'master' && github.event.review.state == 'approved'
+    name: enqueue
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions-ecosystem/action-add-labels@v1
```
Comment (mook-as) [inline .github/workflows/pull-request-queue.yaml:13]: Would it make sense for us to fork this somewhere where we know it won't change under us?  Depending on [some random guy](https://github.com/orgs/actions-ecosystem/people) doesn't seem awesome.

Alternatively, I think [github-script](https://github.com/actions/github-script#apply-a-label-to-an-issue) would work too — and that one is first-party.

Comment (viovanov) [reply .github/workflows/pull-request-queue.yaml:13]: I don't know if it makes sense to fork.
Is this any different from depending on something like [1] or [2]?

[1] https://github.com/bmatcuk/doublestar
[2] https://github.com/FooBarWidget/multipart-parser

Comment (mook-as) [reply .github/workflows/pull-request-queue.yaml:13]: Both of those are libraries (go and C++, respectively), which means they can be sensibly vendored.  In GitHub actions that really isn't a thing (though we _could_ pin to a commit; that seems fine too).

Comment (viovanov) [reply .github/workflows/pull-request-queue.yaml:13]: Seems like a slippery slope to assume maliciousness at this stage.
A version can be replaced for go libraries as well, and I would consider that more of a problem than this.

Comment (mook-as) [reply .github/workflows/pull-request-queue.yaml:13]: Go libraries only update when we do some action (bump things in `go.mod`); GitHub Actions versions are pulled when they run.  Note that Actions uses _branch names_, not tags, for versioning.

Comment (viovanov) [reply .github/workflows/pull-request-queue.yaml:13]: Fair enough 👍, I will try pinning to a commit ID.

**Pull request merged**